aptos-move/aptos-gas-meter/src/meter.rs

```diff
@@ -469,6 +470,23 @@ where
 
         self.algebra.charge_execution(cost)
     }
+
+    #[inline]
+    fn charge_dependency(
+        &mut self,
+        _is_new: bool,
+        addr: &AccountAddress,
+        _name: &IdentStr,
+        size: NumBytes,
+    ) -> PartialVMResult<()> {
+        // TODO: How about 0xA550C18?
+        if self.feature_version() >= 14 && !addr.is_special() {
```

Comment:
what's special about the special addresses in this context? (documentation just mentions format)